src/khoj/interface/web/chat.html

```diff
@@ -2039,11 +2060,33 @@
                 console.error("Error loading file filters:", error);
             });
         }
+
+        function inputAutoFiller(key){
+            var chatInput = document.getElementById("chat-input");
```

Comment:
Should we also show a dropdown menu with the chat history here? Similar to the dropdown menu UX of the slash commands?

Reply:
Possibly. I don't know if it makes sense to show so many messages at a time.